### ISSUE TITLE
How to convert .bck to .fbx? And how to load animations.

### ISSUE THREAD

**Issue by MarioGladiator** [niacdoial/blemd#63]
I'm wondering on how to do this I will send the bck and bmd to see if anything is wrong.

**Comment by niacdoial** [comment]
Bck files are the animations themselves?
Can you describe your problem in more detail, please? what you tried, what results you obtained?

**Comment by MarioGladiator** [comment]
I just don't know how to set it up properly can we talk in like Discord about the issue?

**Comment by MarioGladiator** [comment]
User is ʚ🎭ʚ#6364

**Comment by niacdoial** [comment]
I would prefer keeping this discussion here.
what do you mean you can't set it up properly? what's "it"? the entire add-on?

**Comment by MarioGladiator** [comment]
I'm saying, how do you import the animations and get them to play?

**Comment by niacdoial** [comment]
to import the animations, you need to import the model (bmd file) itself, and that will detect all the bck files which are:
- next to the bmd file
- in a folder named `bck`, `bcks` or  `scrn`, next to *the folder containing the bmd file*.
If you look at the dialog where you select the bmd file to import, there are a lot of options on the right, including options on how the animations are imported. (by default, they *are* imported)
If after all that, they don't load, then it's possible the animations were made for another bmd file, with a different amount of bones.

**Comment by MarioGladiator** [comment]
It's not working let me send you the file. Which contains the file.

**Comment by MarioGladiator** [comment]
I mean image. Is this how you lay it out? 
![unknown](https://user-images.githubusercontent.com/115901575/196041712-20a77dbb-859f-4582-baa0-5e7da1287d6d.png)

**Comment by niacdoial** [comment]
yes, it should work that way.

**Comment by MarioGladiator** [comment]
Does the animation go inside the armature so the model and animation are together

**Comment by MarioGladiator** [comment]
And can this be exported to fbx

**Comment by niacdoial** [comment]
yes, the animation is linked to the armature (the "skeleton" of the mesh).
as for if it can be exported to fbx, I think so, but you might need to tweak the settings on the fbx exporter.

**Comment by MarioGladiator** [comment]
How do I tweak them?

**Comment by niacdoial** [comment]
the settings of the fbx exporter? they are shown either to the left or the right of the window in the "save as" dialog while exporting from blender
